### PR TITLE
Perera example

### PR DESCRIPTION
**Pull request by connorcoley**
- Including example of 5760 reactions from https://science.sciencemag.org/content/359/6374/429
- Also fixed validation bugs encountered during testing (because `Boolean.FALSE == 2`)
- Adds `float_value` field for `Data` to better fit processed data relating to peak areas or masses

It takes ~20 seconds to build the 5760 reactions, so let me know if you'd prefer to have fewer run during automated validation.

### CODE REVIEW

File: proto/reaction.proto [hunk 1]
```diff
@@ -916,6 +916,7 @@ message Data {
     string value = 1;
     bytes bytes_value = 2;
     string url = 3;  // URL for data stored elsewhere.
+    float float_value = 6;
```
Comment (skearnes) [inline proto/reaction.proto:919]: I'm ok with adjusting the tag indices here. I think we should prefer to keep it cleaner/more readable until we move out of "alpha" testing.

File: proto/reaction.proto [hunk 2]
```diff
@@ -916,6 +916,7 @@ message Data {
     string value = 1;
```
Comment (skearnes) [inline proto/reaction.proto:916]: Change this to `string_value`?

Comment (connorcoley) [reply proto/reaction.proto]: Fixed by c177aa9; I chose not to also add an `int_value`, but we can do that to cover all of our bases

File: proto/reaction.proto (hunk 1 above)
Comment (connorcoley) [reply proto/reaction.proto]: Fixed by c177aa9

File: ord_schema/validations.py [hunk 3]
```diff
@@ -463,8 +465,11 @@ def validate_reaction_workup(message):
 
 
 def validate_reaction_outcome(message):
+    # pylint: disable=singleton-comparison
     # Can only have one desired product
-    if sum(product.is_desired_product for product in message.products) > 1:
+    if sum(message_helpers.boolean_unconverter(
+            product.is_desired_product) == True
```
[the remark this answers is not in the document]
Comment (connorcoley) [reply ord_schema/validations.py]: Fixed by c177aa9

File: ord_schema/message_helpers.py [hunk 4]
```diff
@@ -22,6 +22,25 @@
...
+    # pylint: disable=singleton-comparison
+    if boolean == True:
+        return reaction_pb2.Boolean.TRUE
+    if boolean == False:
+        return reaction_pb2.Boolean.FALSE
+    return reaction_pb2.Boolean.UNSPECIFIED
+
+
+def boolean_unconverter(boolean_message):
+    """Converts a Boolean message to the equivalent {None, True, False}"""
```
[the remark this answers is not in the document]
Comment (connorcoley) [reply ord_schema/message_helpers.py]: Fixed docstring/variable name in c177aa9

File: ord_schema/message_helpers.py [hunk 5]
```diff
@@ -22,6 +22,25 @@
...
+    """Converts {None, True, False} to the equivalent Boolean message"""
+    # pylint: disable=singleton-comparison
+    if boolean == True:
+        return reaction_pb2.Boolean.TRUE
+    if boolean == False:
+        return reaction_pb2.Boolean.FALSE
+    return reaction_pb2.Boolean.UNSPECIFIED
+
+
+def boolean_unconverter(boolean_message):
```
[the remark this answers is not in the document]
Comment (connorcoley) [reply ord_schema/message_helpers.py]: Fixed in c177aa9

File: ord_schema/message_helpers.py [hunk 6]
```diff
@@ -22,6 +22,25 @@
 # pylint: disable=too-many-branches
 
 
+def boolean_converter(boolean):
```
[the remark this answers is not in the document]
Comment (connorcoley) [reply ord_schema/message_helpers.py]: Fixed in c177aa9

**Pull request merged**